docs/dev/libs/ml/quickstart.md

```diff
@@ -146,7 +145,23 @@ create a classifier.
 
 ## Classification
 
-Once we have imported the dataset we can train a `Predictor` such as a linear SVM classifier.
+After importing the training and test dataset, they need to be prepared for the classification. 
+Because Flink SVM only supports threshold binary values of `+1.0` and `-1.0`, a conversion is 
+needed after loading the LibSVM dataset since it is labelled using `1`s and `0`s.
```

Comment:
Just a wording thing, I would swap `Because` and `since` in this sentence:
**Since** Flink SVM only supports threshold binary values of `+1.0` and `-1.0`, a conversion is 
needed after loading the LibSVM dataset **because** it is labelled using `1`s and `0`s.